microsoft-edge/test-and-automation/test-and-automation.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Test and automation in Microsoft Edge
```

Comment:
@bwalderman would you mind please giving this test automation intro page a quick read. It's supposed to serve as a welcome/intro/landing page for the things we have under the `Test and automation` table of content.

Reply:
@bwalderman - rendered page for review:
https://review.docs.microsoft.com/en-us/microsoft-edge/test-and-automation/test-and-automation?branch=pr-en-us-1665